### ISSUE TITLE
JAWS doesn't read integrated terminal

### ISSUE THREAD

**Issue by MS-AlexB** [microsoft/vscode#144928]
Issue Type: <b>Bug</b>

1. With JAWS running, open latest Insiders build of VS code.
2. Make sure screen-reader optimization is on, Alt+F1, Ctrl+E for permanent optimization.
3. Try to build a project with Ctrl+Shift+B to get an integrated terminal window with some output to open.
4. Press F6 until you reach the integrated terminal area.
5. JAWS will report to use Ctrl+Up/Down to navigate the terminal's buffer. Try to use Ctrl+Up/Down.

Actual: JAWS is silent and does not read the terminal's output.
Expected: Ctrl+Up/Down will read the terminal's output line by line.

VS Code version: Code - Insiders 1.66.0-insider (6b7453421201c888b81eae5c04ec822a6423608f, 2022-03-11T05:17:19.559Z)
OS version: Windows_NT x64 10.0.22000
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i9-8950HK CPU @ 2.90GHz (12 x 2904)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|31.85GB (16.77GB free)|
|Process Argv|C:\\repos\\fabric-internal\\packages\\excel_substrate_logger --crash-reporter-id 55dec8bb-14d6-4c04-87d0-0c4ef8a63220|
|Screen Reader|yes|
|VM|0%|
</details><details><summary>Extensions (2)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-eslint|dba|2.2.2
prettier-vscode|esb|9.3.0

</details><details>
<summary>A/B Experiments</summary>

```
vsliv695:30137379
vsins829:30139715
vsliv368:30146709
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythonvspyl392:30422396
pythontb:30258533
pythonvspyt551cf:30291413
pythonptprofiler:30281269
vshan820:30294714
pythondataviewer:30285072
vscod805cf:30301675
pythonvspyt200:30323110
bridge0708:30335490
bridge0723:30353136
vsaa593cf:30376535
pythonvs932:30404738
wslgetstarted:30449409
vsclayoutctrc:30405799
dsvsc009:30440021
pythonvsnew555:30438690
vscscmwlcmt:30428973
vscgsvid1:30433757
dockersurvey3:30438168
pynewfile477:30451556

```

</details>

**Comment by isidorn** [comment]
@MS-AlexB thanks for opening this issue. 
@TylerLeonhardt @meganrogge Alex is working for MS and we can reach out to him if more details are needed.

This sounds familiar. Are there more issues with JAWS + terminal? Can I help somehow here?

**Comment by DeafblindEngineer** [comment]
Hello. I have just installed the latest versions of Windows 11 22H2, JAWS 2023, NVDA, and VS Code on a laptop.

I can’t read the output on my braille display from the integrated terminal. Normally pressing INSERT+Z in JAWS (to turn off the Virtual Cursor) or INSERT+SPACEBAR in NVDA (focus mode) would let me read line by line when pressing CTRL+Up Arrow Key. But I am not getting the output. Instead, I am reading list numbers such as 45 of 47.
 
Currently to read the output with JAWS I must do this:
 
Press F6 to navigate to the integrated terminal. Then press INSERT+Z, then CTRL+Up Arrow Key, then press the Plus Key on the Numpad twice. I can then read the output. Then I press CTRL+Up Arrow Key to move up a line and double tap the Plus Key on the Numpad to read the output.
 
For NVDA:
 
Press F6 to navigate to the integrated terminal. Press CTRL+Up Arrow Key, then press INSERT+SPACEBAR to read the output. Then press CTRL+Up Arrow Key to move up a line. Then press INSERT+SPACEBAR. Then INSERT+SPACEBAR again to read a line.

I hope this information can helps to resolve the issue.

Paul

**Comment by rperez030** [comment]
@meganrogge  How is this a duplicate of #164988. This is something that used to work well in the passt. #164988 can be considered a feature request because it is asking for something that screen readers do not normally do, which is reading a line when the change in cursor position has not been initiated by the user.

**Comment by Tyriar** [comment]
I think it was meant to be this one: https://github.com/microsoft/vscode/issues/154027

**Comment by meganrogge** [comment]
Yes my mistake @rperez030 , I did mean #154027. I am aware that this is a regression not a feature request and plan to look into it this iteration
